### PR TITLE
Add `utils.Err` helper

### PR DESCRIPTION
**Pull request by dhrubabasu**
## Why this should be merged

We initialize a `wrappers.Err` struct and immediately return the error in a few places. This pattern can be cleaned up via a helper.

## How this works

Adds a `utils.Err` helper and replaces all instances of this pattern with it.

## How this was tested

CI

### CODE REVIEW

File: vms/components/verify/verification.go [hunk 1]
```diff
@@ -32,3 +32,12 @@ func All(verifiables ...Verifiable) error {
 	}
 	return nil
 }
+
+func Err(errors ...error) error {
```
Comment (joshua-kim) [inline vms/components/verify/verification.go:36]: Should we move this to `wrappers`?

Comment (dhrubabasu) [reply vms/components/verify/verification.go:36]: Added it to `utils`

File: genesis/config.go [hunk 2]
```diff
@@ -172,31 +171,32 @@ func init() {
...
 	localConfig, err := unparsedLocalConfig.Parse()
-	errs.Add(err)
-	LocalConfig = localConfig
-
-	if errs.Errored() {
-		panic(errs.Err)
+	if err != nil {
+		panic(err)
 	}
+	LocalConfig = localConfig
```
Comment (StephenButtolph) [inline genesis/config.go:199]:
```go
	MainnetConfig, err = unparsedMainnetConfig.Parse()
	if err != nil {
		panic(err)
	}

	FujiConfig, err = unparsedFujiConfig.Parse()
	if err != nil {
		panic(err)
	}

	LocalConfig, err = unparsedLocalConfig.Parse()
	if err != nil {
		panic(err)
	}
```

File: vms/components/verify/verification.go [hunk 3]
```diff
@@ -32,3 +32,12 @@ func All(verifiables ...Verifiable) error {
...
+	for _, err := range errors {
+		if err != nil {
+			return err
+		}
+	}
+	return nil
+}
```
Comment (StephenButtolph) [inline vms/components/verify/verification.go:43]:
```suggestion
```

**Review by StephenButtolph — APPROVED** [review]

**Review by darioush — APPROVED** [review]

**Pull request merged**